src/NUnitTestExtractor/NUnitTestExtractor/Program.cs

```diff
@@ -0,0 +1,99 @@
+                {
+                    if (_options.Help)
+                    {
+                        Console.WriteLine(CommandLine.Text.HelpText.AutoBuild(_options));
+                        return;
+                    }
+                   
+                    if(_options.Output != null)
+                    {
+                        if (!Directory.Exists(_options.Output))
+                        {
+                            Directory.CreateDirectory(_options.Output);
+                            Console.WriteLine("created directory at {0}", _options.Output);
+                        }
+                    }
+
+                    if (_options.Level != null)
+                    {
+                        try
+                        {
+                            level = (Level)Enum.Parse(typeof(Level), _options.Level, true);
+                            WriteToTxtFile("file", _options.Output, _options.DLLs, level);
+                        }
+                        catch (ArgumentException e)
+                        {
+                            Console.WriteLine("ArgumentException: Please enter either Namespace, Class, or Function for level");
+                        }
+                    }
+                }
+            }
+        }
+        //While reflection is not implemented this method writes path of dll if it exists with specified level to txt file
+        private static void WriteToTxtFile(string name, string outputDirectory, IList<string> dllDirectories, Level level)
+        {
+            //user entered some dlls
+            if (dllDirectories.Count != 0)
+            {
+                if (outputDirectory == null)
+                {
+                    Console.WriteLine("No output directory specified, writing to stdout");
+                    foreach (string dll in dllDirectories)
+                    {
+                        if (File.Exists(dll))
+                        {
+                            Console.Out.WriteLine("{0}|{1}", dll, level);
+                        }
+                        else
+                        {
+                            Console.Out.WriteLine("Error: {0} file cannot be found on system", dll);
+                        }
+                    }
+                }
+                else
+                {
+                    foreach (string dll in dllDirectories)
+                    {
+                        if (File.Exists(dll))
+                        {
+                            File.AppendAllText(outputDirectory + name + ".txt", dll + "|" + level + Environment.NewLine);
+                        }
+                        else
+                        {
+                            Console.WriteLine("Error: {0} file cannot be found on system", dll);
+                        }
+                    }
+                }
+            }
+            else
+            {
+                Console.WriteLine("Error: no dlls specified");
```

Comment:
This should use Console.Error.WriteLine() and then exit -2 (use different negative exit codes to differentiate different failure reasons).